packages/mobile/src/app/ErrorScreen.tsx

```diff
@@ -43,4 +43,14 @@ class ErrorScreen extends React.Component<Props> {
   }
 }
 
+const styles = StyleSheet.create({
+  errorMessage: {
+    ...fontFamily.regular,
+    fontSize: 12,
+    borderRadius: 25,
+    backgroundColor: 'rgba(238, 238, 238, 0.75)',
```

Comment:
We shouldn't redefine the `fontSize` in most cases.
And we should use an existing color from the design system.

Reply:
Agreed, I was just porting over existing styling. I'm reluctant to start changing actual style properties in this PR because the scope is already so large that it would make it difficult to make sure everything was done correctly.

Reply:
A few properties (fontStyle, lineHeight and color) seem to have changed from the snapshot anyways. Looks like these screens have not gone through the redesign yet. We'll probably do that at some point, so either way seems fine to me.